src/Pages/Wizard/Wizard.js

```diff
@@ -17,14 +27,66 @@ const Wizard = () => {
         setStep(step - 1)
     }
 
+    const progressStep = () => {
+        if (step === 1) {
+            return (
+                <span>0</span>
+            )
+        } else if ( step === 2) {
+            return (
+                <span>1</span>
+            )
+        } else if ( step === 3) {
+            return (
+                <span>2</span>
+            )
+        } else {
+            return (
+                <span>3</span>
+            )
+        }
+    }
+
+    const submitForm = (e) => {
+        e.preventDefault();
+        const aplicant = {...data}
+
+        if (data.doesHaveAgricultureSkills === "" || data.agricultureSkills === "") {
+            aplicant.agricultureSkills = "No agriculture skills."
+        } 
+
+        if (data.doesHaveMetalworkSkills === "" || data.metalworkSkills === "") {
+            aplicant.metalworkSkills = "No metal work skills."
+        }
+
+        if (data.convictions[0].forWhat === "" || data.convictions[0].convictionDate === "") {
+            aplicant.convictions[0].forWhat = "Nothing to report";
+            aplicant.convictions[0].convictionDate = "0001-01-01"
+        }
+        const urlForPost = url3 + "/applicants";
+        axios.post(urlForPost, aplicant)
+        .then(response => {
+            if (response.status < 200 && response.status > 400) {
+                throw new Error("Could not fetch the data!")
+            } else {
+                setStep(4);
+                setData(emptyData)
+            }})
+        .catch((err) => {
+        alert(err.message);
+        setError(err)
+        })
+    }
+     
     const switchSections = () => {
         switch (step) {
             case 1: 
-                return (<StepOne nextStep={nextStep} />)
+                return (<StepOne nextStep={nextStep} setData={setData} data={data}/>)
```

Comment:
There is also alternative to switch - please see this [interesting article](https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals).